js/ui/paygUnlockDialog.js

```diff
@@ -193,6 +201,16 @@ var PaygUnlockDialog = new Lang.Class({
         this._buttonBox = this._createButtonsArea();
         promptBox.add_child(this._buttonBox);
 
+        let helpLineMain = new St.Label({ style_class: 'unlock-dialog-payg-help-main',
+                                          text: _("Dont have an unlock code? That's ok!"),
```

Comment:
s/Dont/Don’t/

s/That's/That’s/ (Unicode apostrophe)

s/ok/OK/

Reply:
All addressed